qcodes/instrument_drivers/Keysight/N52xx.py

```diff
@@ -73,6 +77,7 @@ def get_raw(self) -> Sequence[float]:
 
         return data
 
+
```

Comment:
I think the name of the PR is bad. I simply need to subclass the base driver to make a driver for the N5222B. Right now I have simply defined the driver in notebook cell, but it will get its own module.